### PR TITLE
jsonnet: Add jsonnet files to generate manifests

### PR DESCRIPTION
**Pull request by kakkoyun**
Signed-off-by: Kemal Akkoyun <kakkoyun@gmail.com>

### CODE REVIEW

**Review by bwplotka — APPROVED** [review]
Nice, thanks! :muscle: 
LGTM

File: jsonnet/main.jsonnet [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  name: 'observatorium-up',
+  version: 'master-2020-06-03-8a20b4e',
+  image: 'quay.io/observatorium/up:' + cfg.version,
+  replicas: 1,
+  endpointType: 'metrics',
+  writeEndpoint: 'http://FAKE.svc.cluster.local:8080/api/metrics/v1/test/api/v1/receive',
+  readEndpoint: 'http://FAKE.svc.cluster.local:8080/api/metrics/v1/test/api/v1/query',
+};
+
+local up = (import 'lib/up.libsonnet')(commonConfig);
```
Comment (metalmatze) [inline jsonnet/main.jsonnet:13]:
```suggestion
local up = (import './lib/up.libsonnet')(commonConfig);
```

File: jsonnet/main.jsonnet [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+local job = (import 'lib/job/up.libsonnet')(commonConfig { backoffLimit: 5 });
```
Comment (metalmatze) [inline jsonnet/main.jsonnet:14]:
```suggestion
local job = (import './lib/job/up.libsonnet')(commonConfig { backoffLimit: 5 });
```

File: jsonnet/jsonnetfile.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "version": 1,
+  "dependencies": [
+    {
+      "source": {
+        "local": {
+          "directory": "lib"
+        }
+      },
+      "version": "."
+    }
```
Comment (metalmatze) [inline jsonnet/jsonnetfile.json:11]: Definitely remove this local dependency, but probably we don't need this file anymore to begin with.

**Review by metalmatze — CHANGES_REQUESTED** [review]

**Pull request merged**